openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -40,6 +42,21 @@
 # =============================================================================================
 
 
+def normalize_file_format(file_format):
+    return file_format.upper()
+
+
+def _require_text_file_obj(file_obj):
+    try:
+        file_obj.write("")
+    except TypeError:
+        # Switch to a ValueError and use a more informative exception
+        # message to match RDKit's toolkit writer.
+        raise ValueError(
+            "Need a text mode file object like StringIO or a file opened with mode 't'"
```

Comment:
Should this be mode `'r'`?

Reply:
"r" is for "read". "rt" is to read in text mode. "rb" is to read in binary mode. If unspecified, the default is "t", so you rarely see it written explicitly. https://docs.python.org/3/library/functions.html?highlight=text+mode#open